integration-test/src/main/java/com/sequenceiq/it/cloudbreak/cloud/v4/aws/AwsProperties.java

```diff
@@ -83,6 +85,17 @@ public void setGovCloud(Boolean govCloud) {
         this.govCloud = govCloud;
     }
 
+    public Boolean getExternalDatabaseSslEnforcementSupported() {
+        if (externalDatabaseSslEnforcementSupported == null) {
+            return Boolean.FALSE;
+        }
+        return externalDatabaseSslEnforcementSupported;
+    }
```

Comment:
either this should return `boolean` if it can't be null, or null should be valid.
you could simplify this:
```
return Boolean.TRUE.equals(externalDatabaseSslEnforcementSupported);